home.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Web Conference 2021</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+      type="text/css"
+    />
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <header class="header">
+        <nav class="first-nav">
+          <i class="fa fa-times icon cursor xl-none none" id="close"></i>
+          <i class="fa fa-bars icon cursor xl-none" id="open"></i>
+            <ul class="first-nav-ul">
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor"><i class="fa fa-facebook-f"></i></a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor"><i class="fa fa-twitter"></i></a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor">English</a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor">My page</a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor">Logout</a></li>
+            </ul>
+            </nav>
+        <nav class="second-nav none" id="nav">
+            <ul class="second-nav-logo">
+              <li><a href="./home.html"><i class="fa fa-first-order footer-icon"></i></a></li>
+                <li class="second-nav-logo-list">Global Tech 2021</li>
+            </ul>
+            <ul class="second-nav-ul">
+               <li><a href="./about.html" class="second-nav-anchor">About</a></li> 
+               <li><a href="#" class="second-nav-anchor">Program</a></li> 
+               <li><a href="#" class="second-nav-anchor">Join</a></li> 
+               <li><a href="#" class="second-nav-anchor">Sponsor</a></li> 
+               <li><a href="#" class="second-nav-anchor">News</a></li> 
+               <li><a href="#" class="second-nav-anchor last">Academics</a></li> 
+            </ul>
+        </nav>
+    </header>
+      <main>
+       
+          <section class="main-section">
+             <h1 class="main-header m-top">"Hello! Technology World"</h1>
```

Comment:
- [ ] Top menu should be hidden in mobile resolution. 
- [ ] On mobile resolution background of the navigation section should transparent or the same color as the background.

Original Design:
![Screenshot from 2021-07-01 19-52-23](https://user-images.githubusercontent.com/1642014/124161546-372c2b80-daa6-11eb-9d35-8edaf662b592.png)

Yours:
![Screenshot 2021-07-01 at 19-55-12 Web Conference 2021](https://user-images.githubusercontent.com/1642014/124161594-4612de00-daa6-11eb-999c-8224c7763920.png)

Reply:
:x: This is not addressed yet

Reply:
Hi de-scholar,
I have been able to make changes to this, giving the first navbar a display none so it doesn't show when the hamburger isn't clicked and it looks similar to the original design
https://github.com/bimbolabuari/mv-html-capstone/blob/0eeb5bfafd46b7adf08ca9a0f227633c2423443c/home.html#L20
![Screenshot (295)](https://user-images.githubusercontent.com/66112760/124191073-21753100-dabb-11eb-9d8e-cb3ca14e5fe5.png)
I would appreciate any suggestion on what you think I could do about this 👍 
Thanks ❤️

Reply:
This issue is well addressed.